judges/dimensions-of-terrain/dimensions-of-terrain.rb

```diff
@@ -103,3 +103,14 @@
   end
 end
 f.total_contributors = contributors.count
+
+# Total number of unique active contributors to all repos
+active_contributors = Set.new
+Fbe.unmask_repos.each do |repo|
+  Fbe.octo.search_commits(
+    "repo:#{repo} author-date:>#{(f.when - (30 * 24 * 60 * 60)).iso8601[0..9]}"
+  )[:items].each do |commit|
+    active_contributors << commit.dig(:author, :id)
+  end
+end
+f.total_active_contributors = active_contributors.count { |c| !c.nil? }
```

Comment:
@Yegorov maybe it's better use `compact`: `active_contributors.compact.count`
or check for `nil`, when we add values to `active_contributors`?

Reply:
@Suban05 Ok, probably it is better to check at once for null. Thanks, fixed!